### ISSUE TITLE
Running `protontweaks` without any arguments should return the help info

### ISSUE THREAD

**Issue by PoorPocketsMcNewHold** [ribbon-studios/protontweaks#16]
```
thread 'main' panicked at src/commands/run.rs:62:42:
called `Option::unwrap()` on a `None` value
stack backtrace:
   0:     0x55c37e88f666 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h410d4c66be4e37f9
   1:     0x55c37e8bed50 - core::fmt::write::he40921d4802ce2ac
   2:     0x55c37e88bb5f - std::io::Write::write_fmt::h5de5a4e7037c9b20
   3:     0x55c37e88f444 - std::sys_common::backtrace::print::h11c067a88e3bdb22
   4:     0x55c37e890d27 - std::panicking::default_hook::{{closure}}::h8c832ecb03fde8ea
   5:     0x55c37e890a89 - std::panicking::default_hook::h1633e272b4150cf3
   6:     0x55c37e8911b8 - std::panicking::rust_panic_with_hook::hb164d19c0c1e71d4
   7:     0x55c37e891059 - std::panicking::begin_panic_handler::{{closure}}::h0369088c533c20e9
   8:     0x55c37e88fb66 - std::sys_common::backtrace::__rust_end_short_backtrace::hc11d910daf35ac2e
   9:     0x55c37e890de4 - rust_begin_unwind
  10:     0x55c37e144bc5 - core::panicking::panic_fmt::ha6effc2775a0749c
  11:     0x55c37e144c83 - core::panicking::panic::h44790a89027c670f
  12:     0x55c37e144b16 - core::option::unwrap_failed::hcb3a256a9f1ca882
  13:     0x55c37e151296 - protontweaks::commands::run::parse_command::{{closure}}::h22f0033434bf4525
  14:     0x55c37e154076 - protontweaks::commands::handle::{{closure}}::hf9770a0f1ed34e6d
  15:     0x55c37e14d513 - tokio::runtime::park::CachedParkThread::block_on::h0c83b895f030268a
  16:     0x55c37e1b4baa - tokio::runtime::context::runtime::enter_runtime::hb7cfbf48fadcc82c
  17:     0x55c37e1bc7be - tokio::runtime::runtime::Runtime::block_on::h88af6dc8d6c7df2d
  18:     0x55c37e1aa3a9 - protontweaks::main::hb6232d9af6704bac
  19:     0x55c37e1948e3 - std::sys_common::backtrace::__rust_begin_short_backtrace::h6f995e0013e6f0a6
  20:     0x55c37e1b60b9 - std::rt::lang_start::{{closure}}::h2a2c67af66ae4a09
  21:     0x55c37e884091 - std::rt::lang_start_internal::h4d236095b69a230b
  22:     0x55c37e1aa475 - main
  23:     0x7f72d072c088 - __libc_start_call_main
  24:     0x7f72d072c14b - __libc_start_main_impl
  25:     0x55c37e145415 - _start
  26:                0x0 - <unknown>
```
Fedora 40. Tried with the AppImage and .rpm native package. I do have the latest "stable" version of protontricks installed from my main distribution repo.

**Comment by cecilia-sanare** [comment]
**Looks like if you run it with just:**

```sh
$ protontweaks
```

It throws the error you're running into instead of outputting the help information.
I'm assuming that's all you're running to hit this error?

**Comment by PoorPocketsMcNewHold** [comment]
```
 ❯ ./protontweaks.AppImage 
thread 'main' panicked at src/commands/run.rs:62:42:
called `Option::unwrap()` on a `None` value
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
Had the same output with the native .rpm package.
Passing --help does resolve a bunch of the same issues.

**Comment by cecilia-sanare** [comment]
Should be fixed in v0.8.1 once it finishes building!
